### PR TITLE
COMMERCE-4789 making code consistent to calculate prices correctly in all content contributors

### PR DESCRIPTION
**Pull request by igorbeslic**
cc @marco-leo

### CODE REVIEW

**Comment by igorbeslic** [comment]
ci:test:sf

**Comment by igorbeslic** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-frontend-impl/src/main/java/com/liferay/commerce/frontend/internal/util/ProductHelperImpl.java [hunk 1]
```diff
@@ -74,23 +75,55 @@ public PriceModel getMinPriceModel(
...
 		CommerceProductPrice commerceProductPrice =
 			_commerceProductPriceCalculation.getCommerceProductPrice(
-				cpInstanceId, quantity, true, commerceContext);
+				commerceProductPriceRequest);
 
 		if (commerceProductPrice == null) {
 			return null;
 		}
 
+		final CommerceContext commerceContext =
```
Comment (dsitu) [inline modules/apps/commerce/commerce-frontend-impl/src/main/java/com/liferay/commerce/frontend/internal/util/ProductHelperImpl.java:90]: @igorbeslic does this need to be final?

**Review by dsitu — CHANGES_REQUESTED** [review]

**Comment by marco-leo** [comment]
Hey @igorbeslic i can’t merge major version bump in this moment, please avoid it. Thank

**Comment by igorbeslic** [comment]
ci:test:sf

**Comment by igorbeslic** [comment]
ci:test:relative

**Comment by dsitu** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-frontend-impl/src/main/java/com/liferay/commerce/frontend/internal/util/ProductHelperImpl.java (hunk 1 above)
Comment (igorbeslic) [reply modules/apps/commerce/commerce-frontend-impl/src/main/java/com/liferay/commerce/frontend/internal/util/ProductHelperImpl.java]: Tnx!

**Comment by igorbeslic** [comment]
ci:test:relevant

**Comment by igorbeslic** [comment]
ci:stop

**Comment by igorbeslic** [comment]
will close this one - too much troubles

**Pull request closed without merging**